resources/views/profile.blade.php

```diff
@@ -1,394 +1,131 @@
-                            </legend>
-                            <div class="table-responsive">
-
-                                <table class="table table-hover table-condensed">
-                                    <thead>
-                                    <tr>
-                                        <th>From Realm</th>
-                                        <th>Action</th>
-                                        <th>From IP</th>
-                                        <th><span class="glyphicon glyphicon-info-sign pointable" aria-hidden="true"
-                                                  title="Time is on UTC"></span>&nbsp;When
-                                        </th>
-                                    </tr>
-                                    </thead>
-                                    <tbody>
-                                    @foreach ($actions as $action)
-                                        <tr>
-                                            @if(!$action->hasRealm())
-                                                <td>Site</td>
-                                            @else
-                                                <td>{!! $action->getRealm() !!}</td>
-                                            @endif
-                                            <td>{!! $action->getUserAction() !!}</td>
-                                            <td>{!! $action->getFromIp() !!}</td>
-                                            <td>{!! $action->getCreatedAt()->format("Y-m-d H:i:s") !!}</td>
-                                        </tr>
-                                    @endforeach
-                                    </tbody>
-                                </table>
-                            </div>
-                        </div>
-                    </div>
-                @endif
-            </div>
-        </div>
-    </div>
-    <div class="col-md-3">&nbsp;
-    </div>
-@stop
+@append
+@section('meta')
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+@append
+@section('css')
+    {!! HTML::style('assets/css/profile.css') !!}
+@append
 @section('scripts')
-    <script type="application/javascript">
-        $(document).ready(function () {
-            $('#profile', '#main-menu').addClass('active');
+    <script>
+        let trustedSites = [];
+        @foreach ($sites as $site)
+        trustedSites.push({
+            id: "{!! $site->getId() !!}",
+            auth_policy: "{!! $site->getAuthorizationPolicy() !!}",
+            realm: "{!! $site->getRealm() !!}",
+            trusted_data: "{!! $site->getUITrustedData() !!}",
+            deleteURL: "{!! URL::action("UserController@deleteTrustedSite",["id"=>$site->getId()]) !!}"
         });
+        @endforeach
+
+        const initialValues = {
+            address1: '{!! $user->address1 ?? '' !!}',
+            address2: '{!! $user->address2 ?? '' !!}',
+            bio: '{!! $user->bio ?? '' !!}',
```

Comment:
this is breaking bc /r/n sequences
i do think that perhaps we should do the json dump at php side

Reply:
done

Reply:
i do think that probably the cleaner way would be using a serializer from php controller method to serialize to json the initial value of the user object